apps/src/code-studio/assets/SpriteManagementDirectory.jsx

```diff
@@ -11,7 +30,49 @@ export default class SpriteManagementDirectory extends React.Component {
         <h3>
           <a href="/sprites/default_sprites_editor">Edit Default Sprites</a>
         </h3>
+        <div style={styles.pageBreak}>
+          <Button
+            text="Release Changes to Production"
+            color={Button.ButtonColor.red}
+            onClick={() => {
+              let shouldRelease = confirm(
+                'This will release all the sprites you have added and updates to the default sprite list to ' +
+                  'production. Are you sure?'
+              );
+              if (shouldRelease) {
+                this.moveChangesToProduction();
+              }
+            }}
```

Comment:
nit: this `onClick` handler should be in its own function to make the markup more readable